src/style.css

```diff
@@ -1,3 +1,76 @@
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  padding: 0;
+  font-family: Roboto, Arial, sans-serif;
+}
+
+header {
+  background-color: #fff;
+  justify-content: space-between; /* Розподіляє елементи по сторонах */
+  align-items: center; /* Вирівнює елементи по вертикалі в центрі */
+  padding: 0 50px; /* Додаємо внутрішні відступи, якщо потрібно */
+  display: flex;
+  box-sizing: border-box;
+}
+
+.nav__item {
+  margin-right: 20px; /* Відступи між посиланнями */
+}
+
+.nav__item:last-child {
+  margin-right: 0;
+}
+
+.logo {
+  display: block;
+  width: 40px;
+  height: 40px;
+}
+
+.nav {
+  display: flex;
+}
+
+ul {
+  margin: 0;
+  padding: 0;
+  list-style-type: none;
+  display: flex;
+}
```

Comment:
Don't use tag names for styling (except `html` and `body`). Style all elements using `.class` and if needed with `:pseudo-class`, `pseudo-element` and `[attribute]`. For example, instead of styling `ul`, use `.nav__list`.